docs/bigquery/snippets.py

```diff
@@ -697,6 +697,48 @@ def test_client_query(client):
     # [END client_query]
 
 
+def test_client_query_destination_table(client, to_delete):
+    """Run a query"""
+    dataset_id = 'query_destination_table_{}'.format(_millis())
+    dataset_ref = client.dataset(dataset_id)
+    to_delete.append(dataset_ref)
+    dataset = client.create_dataset(bigquery.Dataset(dataset_ref))
```

Comment:
@tswast `flake8` is sad about this unused variable (`dataset`)

Reply:
Ah, oops. I bumped the to_delete up and forgot to remove that. It can go.